content/tutorials/manage-the-rippled-server/configuration/configure-grpc.md

```diff
@@ -0,0 +1,55 @@
+# Configure gRPC
+
+The `rippled` server has an experimental binary API compatible with the [gRPC standard](https://grpc.io/). Currently, this API provides a subset of the full [`rippled` API](rippled-api.html). You can enable the gRPC API on your server with a new configuration stanza. [New in: rippled 1.5.0][]
```

Comment:
```suggestion
The `rippled` server has an experimental [gRPC](https://grpc.io/) API. Currently, this API provides a subset of the full [`rippled` API](rippled-api.html). You can enable the gRPC API on your server with a new configuration stanza. [New in: rippled 1.5.0][]
```
I wouldn't call it a binary API. gRPC doesn't explicitly call it a binary API, and a lot of the methods in our gRPC API have a binary flag, just as the JSON API does. I think calling it a binary API while simultaneously having binary flags in the API is confusing.
I think the previous phrasing of the sentence makes it seem like we rolled our own tech solution that is compatible with gRPC, but in actuality we are running an actual gRPC server within rippled.

Also, I wonder if it would be useful to spell out what subset of the API that we support via gRPC. We support tx, account_tx, submit, fee and account_info. What do you think?

Reply:
I'll rephrase the first part.

This document is not the right place to list what subset of the API is in gRPC. The gRPC overview doc (still forthcoming) will be the right place for that.